ee/localserver/request-id.go

```diff
@@ -68,6 +72,16 @@ func (ls *localServer) requestIdHandlerFunc(res http.ResponseWriter, req *http.R
 	}
 	response.identifiers = ls.identifiers
 
+	consoleUsers, err := consoleUsers()
+	if err != nil {
```

Comment:
I think this is correct for now, but I wonder if we'll want to think about how to expose errors to k2 through this interface. Not today!

Reply:
👍